code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -644,22 +647,109 @@
+	icon_state = "wintercoat_chef"
+	inhand_icon_state = "wintercoat_chef"
+	allowed = list(/obj/item/reagent_containers/spray/plantbgone, /obj/item/plant_analyzer, /obj/item/seeds, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/cultivator, /obj/item/reagent_containers/spray/pestspray, /obj/item/hatchet, /obj/item/storage/bag/plants, /obj/item/graft, /obj/item/secateurs, /obj/item/geneshears)
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/chef
+
+/obj/item/clothing/head/hooded/winterhood/chef
+	desc = "A hood just as clean as the main body, smells of oil anyways."
+	icon_state = "winterhood_chef"
+
+/obj/item/clothing/suit/hooded/wintercoat/mime
+	name = "\improper mime's winter coat"
+	desc = "A black and white striped coat that the mimes made for themselves, who knows what this can do..."
+	icon_state = "wintercoat_mime"
+	inhand_icon_state = "wintercoat_mime"
+	armor = list("melee" = 20, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/mime
+
+/obj/item/clothing/head/hooded/winterhood/mime
+	desc = "A black and white hood that is slightly disorenting to look at, so you try not to look at it."
+	icon_state = "winterhood_mime"
+
+/obj/item/clothing/suit/hooded/wintercoat/clown
+	name = "\improper clown's winter coat"
+	desc = "A wintercoat made specially made for the clowns from an unknown place. All clowns have one, but not all of them wear it."
+	icon_state = "wintercoat_clown"
+	inhand_icon_state = "wintercoat_clown"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/clown
+
+/obj/item/clothing/head/hooded/winterhood/clown
+	desc = "A bright yellow and red hood that screams honk just by looking at it, tends to temporarily blind moths with just a glance."
+	icon_state = "winterhood_clown"
+
+/obj/item/clothing/suit/hooded/wintercoat/chap
+	name = "\improper chaplins's winter coat"
+	desc = "Warm, Stylish, approved by the space pope. Everything you need in a winter coat."
+	icon_state = "wintercoat_chap"
+	inhand_icon_state = "wintercoat_chap"
+	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic, /obj/item/soap, /obj/item/sensor_device, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
```

Comment:
let this hold null rods and the other things that subtypes of /obj/item/clothing/suit/hooded/chaplainsuit can hold, not... whatever this copy+pasted list is.